distribution/lib/Standard/Base/0.0.0-dev/src/Data/Index_Sub_Range.enso

```diff
@@ -1,5 +1,6 @@
 import project.Data.Any.Any
 import project.Data.Numbers.Integer
+import project.Data.Range as Range_Module
```

Comment:
@jdunkerley something to keep in mind in further refactorings (this is because of `up_to`)